app/views/statement.erb

```diff
@@ -1,3 +1,27 @@
- <h1><%= statement.prefLabel.first %></h1>
- <h2><%= statement.rdf_subject %></h2>
- <p><%= statement.definition.first %></p>
+<html>
```

Comment:
:+1: we'll want to move some of this into a `:layout`, but I'm good with pulling it in as-is for now.